### PR TITLE
Feature/sanitize

### PR DESCRIPTION
**Pull request by eliciusc**
[I created the Sanitize component but I'm not sure it's properly implemented.

### CODE REVIEW

File: components/Hero/Hero.jsx [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
                 <div className='body1'>
-                    <p className={styles.intro_text}>{description}</p>
+                    <p className={styles.intro_text}>
+                        <Sanitize input={description} />
+                    </p>
                 </div>
             </div>
             <div className={styles.crop_img_banner}>
-                <Image className={styles.img_banner} alt={image.alt} {...image} />
+                <Sanitize input={image.src} />
```
Comment (marceloglacial) [inline components/Hero/Hero.jsx:23]: you don't need to use on all the props. 
just on the content ones :)

File: components/Sanitize/Sanitize.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import sanitizeHtml from 'sanitize-html'
```
Comment (marceloglacial) [inline components/Sanitize/Sanitize.jsx:1]: Please use a semantical name like: <TextContent>

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: components/Hero/Hero.jsx (hunk 1 above)
Comment (DercilioFontes) [reply components/Hero/Hero.jsx:23]: The Image component needs all props. @eliciusc, you can sanitize inside the Image component. Just alt and src.

Comment (eliciusc) [reply components/Hero/Hero.jsx]: sanitizing on Image component

File: components/Sanitize/Sanitize.jsx (hunk 2 above)
Comment (eliciusc) [reply components/Sanitize/Sanitize.jsx]: changed to sanitize

**Review by marceloglacial — APPROVED** [review]

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**